ai-platform-unified/notebooks/notebook_template.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>import sys, os -&gt;</p><p> </p><p>import sys</p><p>import os</p><p> </p><p>You have an indentation error after if not os.path.exists('/opt/deeplearning/metadata/env_version'):</p><p>The code after it needs to be indented one level</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/251/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Fixed</p>